packages/create-fractal/README.md

```diff
@@ -0,0 +1,18 @@
+# @frctl/create-fractal
+
+New Fractal pattern library initializer.
+
+[![NPM Version](https://img.shields.io/npm/v/@frctl/create-fractal)](https://www.npmjs.com/package/@frctl/create-fractal)
+
+## Usage
+
+Run `npm init @frctl/fractal [dir]` in your terminal. `[dir]` is the new folder you want to create your project in.
```

Comment:
We could also rename the package to just allow `npm init @frctl` or do something longer, e.g. `npm init @frctl/library`, though I'm not sure it would be better. Did not check if we could do just `npm init fractal`, e.g. if the unscoped package name is free on npm.